xtevent/xtevent.sthlp

```diff
@@ -316,6 +342,12 @@ that two-way clustering is allowed with {help reghdfe}.
 {hline}
 
 {pstd}Interaction Weighted Estimator proposed by Sun and Abraham (2021){p_end}
+{phang2}{cmd:. {stata xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure, policyvar(union2) window(3) impute(stag) vce(cluster idcode) sunabraham}}
+{p_end}
+
+{hline}
+
+{pstd}Interaction Weighted Estimator proposed by Sun and Abraham (2021) with user-created cohort variables{p_end}

```

Comment:
@jorpppp 
I tested the examples in the help file. I compared the same example but using
- `sunabraham` option
- option `cohort(create)`
- input variables by the user

To do so, I ran the following code:
```
use nlswork, clear
by idcode (year): gen time=_n
xtset idcode time
by idcode (time): gen union2 = sum(union)
replace union2 = 1 if union2 > 1

*generate the variable that indicates cohort
gen timet=year if union2==1
by idcode: egen time_of_treat=min(timet)
*Generate the variable that indicates the control cohort. 
gen never_treat=time_of_treat==.

* Estimate

xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure, policyvar(union2) window(3) impute(stag) vce(cluster idcode) sunabraham 
est store sunabraham 

xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure, policyvar(union2) window(3) impute(stag) vce(cluster idcode) cohort(create, save replace) 
est store create

xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure, policyvar(union2) window(3) impute(stag) vce(cluster idcode) cohort(variable time_of_treat) control_cohort(variable never_treat)
est store given 

esttab sunabraham create given, mtitles
```

I got these [results](https://github.com/JMSLab/xtevent/assets/36095845/8f3b5f71-655e-47d4-8e80-ea295f0c0768). `sunabraham` and `cohort(create)` produces identical results but I'm getting different results with the manually created variables. I looked at the code and I think `xtevent` generates the variables as I manually did. 
https://github.com/JMSLab/xtevent/blob/1181b94d0f162c04b12bc32466869163e0b49570/xtevent/_eventols.ado#L283

Reply:
@Constantino-Carreto-Romero Thanks, let me check this.

Reply:
@Constantino-Carreto-Romero there are two issues. One is that in the above code you used the "year" variable to create the cohorts but you should have used the "time" variable. That solves most of the discrepancies.

That said, there are still some discrepancies between the cohort variable generated by `xtevent` and the manually generated one. This is because the cohort variable is generated in the `touse` sample that excludes observations that have missing values in the control variables. We are ignoring missing values in the control variables when we generate event-time dummies in `_eventgenvars`, so I think we should ignore them here too. I'll implement this change.

Reply:
Solved in https://github.com/JMSLab/xtevent/pull/185/commits/e65db8df6e74fe05f0b712bc326b65ad64255a78